tests/cpp/test_memory.cpp

```diff
@@ -1061,6 +1061,60 @@ TEST_F(TMAMiscTest, Repro1977) {
   testValidate(&fusion, cg_outputs, {t0}, {t0}, __LINE__, __FILE__);
 }
 
+TEST_F(TMAMiscTest, LoadStrongCorrectness) {
```

Comment:
Can you explain what you'd want to demonstrate/test here? Since the strong correctness is not guaranteed, the hole is filled with the invalid data. Isn't it expected?

Reply:
In the future, I will modify our predication elimination to get strong correctness. I am just adding a test for it before I start the work. When that work is done, this test will be testing that holes are filled with zero (so the line below TODO will be removed)